aten/src/ATen/native/mkldnn/Matmul.cpp

```diff
@@ -78,6 +78,13 @@ bool use_mkldnn_matmul(
     return false;
 }
 
+void mkldnn_matmul_i8i8i32(
+    const Tensor &mat1,
+    const Tensor &mat2,
+    const Tensor &result) {
+  TORCH_CHECK(false, __func__, ": ATen not compiled with MKLDNN support");
```

Comment:
```suggestion
  TORCH_INTERNAL_ASSERT(false, __func__, ": ATen not compiled with MKLDNN support");
```
so that the error user is better.

Reply:
Thanks. Changed.